CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 12.0.0 - May 17, 2024
+
+- Update MSRV to Rust `v1.56.1` [#639](https://github.com/rust-bitcoin/rust-miniscript/pull/639)
+- Drop the `Property` trait entirely [#652](https://github.com/rust-bitcoin/rust-miniscript/pull/652)
+- Improve compiler logic when deciding between conjunctions and `multi`/`multi_a` [#657](https://github.com/rust-bitcoin/rust-miniscript/pull/657)
+- Several locktime improvements [#654](https://github.com/rust-bitcoin/rust-miniscript/pull/654)
+- Derive `Hash` for `pub` items [#659](https://github.com/rust-bitcoin/rust-miniscript/pull/659)
+- Introduce `Threshold` type [#660](https://github.com/rust-bitcoin/rust-miniscript/pull/660), 
```

Comment:
In 7ddaaf6f7f0d91e83eca659ccac340c9cda24883:

Trailing whitespace, and also a trailing comma.

Reply:
Fixed.

\me Puts coin in money jar for not reading diff before pushing

Reply:
That jar is getting full... Reminds me a calculus teacher that had a jar of coins for everytime that a student forgot to add $c$ to an integration procedure 😂

Reply:
I had a swear jar once but it got full so quick I stopped. Now we do 5 pushups each time instead - works way better.

Reply:
Ha! Just did five push ups and said "read your diff" at the top of each one - lets see how it goes.